src/test/java/org/minions/devfund/richard/SquareTest.java

```diff
@@ -0,0 +1,295 @@
+package org.minions.devfund.richard;
+
+import org.junit.Test;
+import org.minions.devfund.richard.square.Square;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class.
+ */
+public class SquareTest {
+
+    private static final int ONE = 1;
+    private static final int TWO = 2;
+    private static final int THREE = 3;
+    private static final int FOUR = 4;
+    private static final int FIVE = 5;
+
+    /**
+     *
+     */
+    @Test
+    public void testPrintMethod() {
+        Square mySquare = new Square(FOUR);
+        final String expectedValue = "1 2 3 4 "
+                + "5 6 7 8 "
+                + "9 10 11 12 "
+                + "13 14 15 16 ";
+        assertEquals(expectedValue, mySquare.print());
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testUpsideDownFlipMethod() {
+        Square mySquare = new Square(FOUR);
+        final String expectedValue = "13 14 15 16 "
+                + "9 6 7 12 "
+                + "5 10 11 8 "
+                + "1 2 3 4 ";
+        assertEquals(expectedValue, mySquare.upsideDownFlip(ONE).print());
```

Comment:
Please Consider using arrays for the assertions and not use the print method.

Reply:
I used array in other tests, this test is specific for print method.